<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="backGround">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>1200</width>
     <height>650</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/firstLevel/photo/playingBackground.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bigGold1">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>420</y>
     <width>130</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/golds/photo/bigGold.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bigGold2">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>450</y>
     <width>130</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/golds/photo/bigGold.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="luckBag1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>580</y>
     <width>70</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/luckyBag/photo/luckyBag.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pig1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/GoldenMiner/photo/pig.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pig2">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>280</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/GoldenMiner/photo/pig.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="stone1">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>430</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/stone/photo/stones.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pigWithDiamond">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>690</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/GoldenMiner/photo/daimondpig.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="diamond1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>540</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/GoldenMiner/photo/daimond.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="smallGold1">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>350</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/golds/photo/smallGold.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="smallGold2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>570</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/golds/photo/smallGold.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="smallGold3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>380</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/golds/photo/smallGold.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="smallGold4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/golds/photo/smallGold.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="stone2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/stone/photo/stones.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="luckBag2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>610</y>
     <width>70</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/luckyBag/photo/luckyBag.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Miner">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>0</y>
     <width>250</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/oldMan/photo/oldMan.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="currentMoney">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前金额：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="showCurrentMoney">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>121</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>$0</string>
   </property>
  </widget>
  <widget class="QLabel" name="goalMoney">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>250</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>目标金额：$1150</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="currentBomb">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>30</y>
     <width>121</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>炸弹数：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="bomb">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>30</y>
     <width>60</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lastingtime">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>20</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>剩余时间：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="remainingTime">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>20</y>
     <width>60</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>60</string>
   </property>
  </widget>
  <widget class="QLabel" name="secondLevel">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>70</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>第三关</string>
   </property>
  </widget>
  <widget class="myWidget" name="youwangWidget" native="true">
   <property name="geometry">
    <rect>
     <x>507</x>
     <y>108</y>
     <width>146</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="drawline" name="invisibleListWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>1200</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
  </widget>
  <widget class="QLabel" name="fail">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="addNumber">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 158, 54);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>backGround</zorder>
  <zorder>bigGold1</zorder>
  <zorder>bigGold2</zorder>
  <zorder>luckBag1</zorder>
  <zorder>pig1</zorder>
  <zorder>pig2</zorder>
  <zorder>stone1</zorder>
  <zorder>pigWithDiamond</zorder>
  <zorder>diamond1</zorder>
  <zorder>smallGold1</zorder>
  <zorder>smallGold2</zorder>
  <zorder>smallGold3</zorder>
  <zorder>smallGold4</zorder>
  <zorder>stone2</zorder>
  <zorder>luckBag2</zorder>
  <zorder>Miner</zorder>
  <zorder>currentMoney</zorder>
  <zorder>showCurrentMoney</zorder>
  <zorder>goalMoney</zorder>
  <zorder>currentBomb</zorder>
  <zorder>bomb</zorder>
  <zorder>lastingtime</zorder>
  <zorder>remainingTime</zorder>
  <zorder>secondLevel</zorder>
  <zorder>invisibleListWidget</zorder>
  <zorder>youwangWidget</zorder>
  <zorder>fail</zorder>
  <zorder>addNumber</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>drawline</class>
   <extends>QListWidget</extends>
   <header>drawline.h</header>
  </customwidget>
  <customwidget>
   <class>myWidget</class>
   <extends>QWidget</extends>
   <header location="global">mywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
